tmt/steps/execute/internal.py

```diff
@@ -10,25 +9,46 @@
 import tmt
 import tmt.utils
 from tmt.steps.execute import TEST_OUTPUT_FILENAME
+from tmt.steps.provision import DEFAULT_RSYNC_OPTIONS
 from tmt.steps.provision.local import GuestLocal
 
-REBOOT_VARIABLES = [
+REBOOT_VARIABLES = (
     "TMT_REBOOT_COUNT",
     "REBOOTCOUNT",
-    "RSTRNT_REBOOTCOUNT",
-    ]
-REBOOT_TYPE = "reboot"
+    "RSTRNT_REBOOTCOUNT")
 REBOOT_SCRIPT_PATHS = (
     "/usr/local/bin/rstrnt-reboot",
     "/usr/local/bin/rhts-reboot",
     "/usr/local/bin/tmt-reboot")
-REBOOT_BACKUP_EXT = ".backup"
+REBOOT_REQUEST_FILENAME = "reboot_request"
 REBOOT_SCRIPT = f"""\
 #!/bin/sh
-echo "{{token}}:{{{{\
-    \\"type\\": \\"{REBOOT_TYPE}\\",\
-    \\"version\\": \\"0.2\\"\
-}}}}"
+touch "$TMT_TEST_DATA/{REBOOT_REQUEST_FILENAME}"
+"""
+REBOOT_BACKUP_EXT = ".backup"
```

Comment:
```suggestion
REBOOT_BACKUP_EXT = ".tmt.backup"
```

Maybe? Just to make it obvious that `tmt` created the backup?